tests/pipeline/resources/streams-bootstrap/defaults.yaml

```diff
@@ -5,9 +5,12 @@ streams-bootstrap:
       schemaRegistryUrl: ${config.schema_registry.url}
   version: "3.0.1"
 
-producer-app: {} # inherits from streams-bootstrap
+producer-app: {
+  image: "producerApp"
```

Comment:
why do we need to set this now?

Reply:
I enforced the `image` field now. I can make it optional, but I cannot imagine a deployment/job/StatefulSet without an image. SB Helm charts default to [`producerApp`](https://github.com/bakdata/streams-bootstrap/blob/master/charts/producer-app/values.yaml#L3) and [`streamsApp`](https://github.com/bakdata/streams-bootstrap/blob/master/charts/streams-app/values.yaml#L5) for the `image` field, which I also think is very strange.

Reply:
yeah this looks odd for sure. Can we at least set it as default on the Pydantic model so that we don't need it in defaults.yaml?

Reply:
We don't need to set it in the defaults as the `image.` In our use cases with set the image on every pipeline entry in the `pipeline.yaml`. I have removed it from the defaults. Have a look at the [pipeline.yaml in.the test folder](https://github.com/bakdata/kpops/blob/18bd7a763c82d29a9782df2eb90fa24ec71a1cbc/tests/pipeline/resources/streams-bootstrap/pipeline.yaml).